.github/workflows/build.yml

```diff
@@ -18,20 +18,13 @@ jobs:
 
     steps:
     - name: Checkout
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
 
     - name: Set up Python
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
 
-    - name: Install dependencies
-      run: pip install toml pep517 packaging importlib_metadata
-
-    - name: Install python2 dependencies
-      if: ${{ matrix.python == 2.7 }}
-      run: pip install typing
-
-    - name: Install build backend
-      run: pip install setuptools wheel
+    - name: install build
```

Comment:
I generally only capitalize stuff that wants to be a paragraph and ends with a period. This does not so IMHO it's more accurate to keep it lower case. My 2c

Reply:
Well, we can discuss the style in a separate issue/PR.

Reply:
Thought as much, hence why only changed the lines I needed to touch to fix the breaking CI.

Reply:
You should still adhere to the style in use.

FYI, I want the first word of the name capitalized because that's the style of the default non-optional CI steps in github actions. With your change we now have one step that does not follow the style.

https://github.com/FFY00/python-build/pull/49/checks?check_run_id=802831513

Reply:
I guess I'm challenging here whoever made Github names by default 🤷